### PR TITLE
SU-3227 Expose townsend score.

### PR DESCRIPTION
**Pull request by aerocov**

### CODE REVIEW

**Review by sesh — APPROVED** [review]
👍

File: Readme.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ const clinical = inputBuilder.buildClinical(
     false,
     false
 );
-const qriskInput = inputBuilder.buildQriskInput(aboutYou, clinical, biometric);
+const qriskInput = inputBuilder.buildQriskInput(aboutYou, clinical, biometric, 0);
```
Comment (sesh) [inline Readme.md]: All the other inputs are named variables, might be nice to be consistent.

Comment (aerocov) [reply Readme.md]: sure

**Review by andrewcaluzzi — APPROVED** [review]
LGTM 😄

**Pull request merged**